modules/ROOT/pages/platforms.adoc

```diff
@@ -0,0 +1,31 @@
+= Platforms
+
+Fedora CoreOS is provisioned via prebuilt disk images, and configured on first-boot via https://github.com/coreos/ignition[Ignition]. Each platform may require specific logic and components, thus dedicated images are provided for each supported environment. Additionally, a unique platform ID is available in the host environment for runtime introspection.
+
+== Runtime introspection
+
+Each Fedora CoreOS image boots with a platform-specific identifier, available on the kernel command-line. The name of the parameter is `ignition.platform.id` and the list of supported platform is documented here.
+
+Platform ID can be introspected at runtime, as follows:
+
+.CLI example of platform introspection
+[source, bash]
+----
+$ grep -o ignition.platform.id='[[:alpha:]]*' /proc/cmdline
```

Comment:
We had ec2 in the past and I think having numbers in platform id's isn't inherently bad, so maybe `alnum` instead of `alpha`?